Cargo.toml

```diff
@@ -36,6 +36,7 @@ targets = [
     "aarch64-unknown-linux-gnu",
     "x86_64-apple-darwin",
     "x86_64-unknown-linux-gnu",
+    "x86_64-pc-windows-gnu",
```

Comment:
I don't know if you want this change or not, happy to remove.

Reply:
I'm not sure if this will demand that we also run a Windows-based CI runner, which we can setup but probably don't need to do here.